lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc

```diff
@@ -325,22 +325,22 @@ function dosomething_helpers_add_js_onboarding($nid) {
       'slides' => [],
     ];
 
-    if (variable_get('dosomething_campaign_onboarding_context_slide', FALSE)) {
-      $onboardingSettings['slides'][] = 'ContextSlide';
+    if (variable_get('dosomething_campaign_onboarding_reportback_items_slide', FALSE)) {
+      $onboardingSettings['slides'][] = 'ReportbackItemsSlide';
     }
 
     if (variable_get('dosomething_campaign_onboarding_instruction_slide', FALSE)) {
       $onboardingSettings['slides'][] = 'InstructionSlide';
     }
 
-    if (variable_get('dosomething_campaign_onboarding_reportback_items_slide', FALSE)) {
-      $onboardingSettings['slides'][] = 'ReportbackItemsSlide';
-    }
-
     if (variable_get('dosomething_campaign_onboarding_campaign_slide', FALSE)) {
       $onboardingSettings['slides'][] = 'CampaignSlide';
     }
 
+    if (variable_get('dosomething_campaign_onboarding_context_slide', FALSE)) {
+      $onboardingSettings['slides'][] = 'ContextSlide';
+    }
+
```

Comment:
Oh gotcha! I thought it worked for me, but maybe that was also cause I might have needed to maybe comment some out and it was just a coincidence it worked right. I feel there's a better way than shuffling code around like that, lol. But ok.